README.md

```diff
@@ -13,6 +13,13 @@ enables remote device management using
 ## Supported Operating System
 
 At the moment only Linux-based systems are supported.
+On Linux system if `edgehog-device-runtime` is a `systemd` service, it can notify `systemd` its status changes.
+This is provided via `rust-systemd` crate that is a rust interface to `libsystemd/libelogind` APIs.
+To build the `runtime` make sure you have `libsystemd-dev` installed on your system
+and the systemd feature enabled.
```

Comment:
Note that if you want to split the paragraphs, you have to leave an empty line between them. If you just go to a newline, the resulting Markdown will put everything on a single line.